roles/variables/vars/main.yml

```diff
@@ -155,11 +155,7 @@ kafka_controller_properties:
       listener.name.controller.plain.sasl.jaas.config: "{{plain_jaas_config}}"
       sasl.mechanism.controller.protocol: PLAIN
   ssl:
-    enabled: "{{
-    kafka_controller_listeners | confluent.platform.ssl_required(kafka_controller_ssl_enabled) or
-    mds_broker_listener.ssl_enabled|bool or
-    mds_tls_enabled|bool or
-    ( kafka_controller_schema_validation_enabled and 'schema_registry' in groups and schema_registry_ssl_enabled ) }}"
+    enabled: "{{ kafka_controller_listeners | confluent.platform.ssl_required(kafka_controller_ssl_enabled) }}"
```

Comment:
won't we need the other conditions once mds is supported?

Reply:
yeah, you are right. I wrote this code when I was under the impression that `mds` would not be supported in the near future.
But that turned up not very accurate.
But in either case, `mds` service would be running on the broker and not on the controller. So this condition will not be used as it is even in the future.